next.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = () => {
   const plugins = [withMarkdoc, withTM]
 
   return plugins.reduce((acc, next) => next(acc), {
-    reactStrictMode: true,
+    reactStrictMode: false,
```

Comment:
Good, I like that. I love what strict mode puts on the table but I hate the double effects calls